qiskit/cluster.py

```diff
@@ -41,8 +56,9 @@ def teleport_to(self, to_processor):
         self.qc.z(to_processor.entanglement_reg).c_if(self.measure_reg[0], 1)
         self.qc.swap(to_processor.entanglement_reg, to_processor.teleport_reg)
 
-    def distributed_controlled_phase(self, angle, control_qubit_index, target_processor,
+    def distributed_controlled_phase_tp(self, angle, control_qubit_index, target_processor,
```

Comment:
What does _tp stand for? Perhaps a more clear name?

Reply:
tp stands for teleport. I wanted to avoid making the function name too long, but if that isn't clear it should be teleport.